doc/conf.py

```diff
@@ -733,8 +734,8 @@ def append_attr_meth_examples(app, what, name, obj, options, lines):
              alt='Clusters'),
         dict(title='Connectivity',
              text='All-to-all spectral and effective connec\u00ADtivity measures.',  # noqa E501
-             url='auto_examples/connectivity/mne_inverse_label_connectivity.html',  # noqa E501
-             img='sphx_glr_mne_inverse_label_connectivity_001.png',
+             url='https://mne.tools/mne-connectivity/stable/auto_examples/mne_inverse_label_connectivity.html',  # noqa E501
+             img='_static/temp_homepage_connectivity.png',  # noqa E501
```

Comment:
This should be changed to a `https://mne.tools/mne-connectivity/stable/_images/...` URL